tests/chaos/oadp/conftest.py

```
@@ -157,10 +160,216 @@ def backup_with_pod_deletion_orchestration(
+
+
+@pytest.fixture()
+def deleted_chaos_namespace(chaos_namespace, deleted_namespace, cleanup_chaos_vms):
+    """
+    Specialized fixture to delete the chaos namespace using the general fixture.
+    """
+    namespace_name = chaos_namespace.name
+    deleted_namespace(namespace_name=namespace_name)
+
+
+@pytest.fixture()
```

Ruff (0.15.0)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)